components/pages/error-page/error-page.composition.tsx

```diff
@@ -1,21 +1,12 @@
 import React from 'react';
-import { ThemeCompositions } from '@teambit/documenter.theme.theme-compositions';
 import { ErrorPage } from './error-page';
 
 export const NotFound = () => {
-  return (
-    <ThemeCompositions>
-      <div style={{ display: 'inline-flex', alignItems: 'center' }}>
-        <ErrorPage code={404} title="Page not found" />
-      </div>
-    </ThemeCompositions>
-  );
+  return <ErrorPage code={404} title="Page not found" style={{ display: 'inline-flex', alignItems: 'center' }} />;
```

Comment:
make sure the composition didn't break

Reply:
All the changes I made, I made them in live mode and I already checked that everything is working properly including compositions and docs.